front-end/src/views/Login.vue

```diff
@@ -1,15 +1,94 @@
-<script setup>
-
-
-</script>
-
 <template>
-  <div id="login">
-    <h1>Login</h1>
-    <router-link to = "/">Home</router-link>
+  <div>
+    <div id="login">
+    <h1>Login</h1>      
+    
+  </div>
+    <form>
+      <label for="username">Username:</label>
+      <input type="text" id="username" v-model="username" />
+      <br />
+      <label for="password">Password:</label>
+      <input type="password" id="password" v-model="password" />
+      <br />
+      <button type="button" @click="login">Login</button>
+    </form>
+    </div>
+    <div class="regLink">
+    Don't have an account? 
+    <router-link to = "/Register">Register here<br></router-link>
+    <br>
+    <router-link to = "/" class="home">Home</router-link>
   </div>
 </template>
 
-<style scoped>
+<script>
```

Comment:
See the conversation below on `Register.vue`. Please change this javascript to Vue's Composition API style

Reply:
okay i will change it.